build_tools/benchmarks/run_benchmarks_on_linux.py

```diff
@@ -38,48 +43,69 @@ def run_benchmark_case(self, benchmark_case: BenchmarkCase,
                          benchmark_results_filename: Optional[pathlib.Path],
                          capture_filename: Optional[pathlib.Path]) -> None:
 
-    # TODO(pzread): Taskset should be derived from CPU topology.
-    # Only use the low 8 cores.
-    taskset = "0xFF"
-
-    # TODO(#11076): Support run_config.
-    if benchmark_case.benchmark_case_dir is None:
-      raise ValueError("benchmark_case_dir can't be None.")
-
-    run_flags = self.__parse_flagfile(benchmark_case.benchmark_case_dir)
-    # Replace the CUDA device flag with the specified GPU.
-    if benchmark_case.driver_info.driver_name == "cuda":
-      run_flags = list(
-          filter(lambda flag: not flag.startswith("--device"), run_flags))
-      run_flags.append(f"--device=cuda://{self.gpu_id}")
-
     if benchmark_results_filename:
       self.__run_benchmark(benchmark_case=benchmark_case,
-                           results_filename=benchmark_results_filename,
-                           taskset=taskset,
-                           run_flags=run_flags)
+                           results_filename=benchmark_results_filename)
 
     if capture_filename:
       self.__run_capture(benchmark_case=benchmark_case,
-                         capture_filename=capture_filename,
-                         taskset=taskset,
-                         run_flags=run_flags)
+                         capture_filename=capture_filename)
 
   def __parse_flagfile(self, case_dir: pathlib.Path) -> List[str]:
     return [
         line.strip()
         for line in (case_dir / MODEL_FLAGFILE_NAME).read_text().splitlines()
     ]
 
+  def __build_tool_cmds(self, benchmark_case: BenchmarkCase,
```

Comment:
Double underscore dunders are discouraged by Google style: https://google.github.io/styleguide/pyguide.html#3162-naming-conventions. Looks like you already have some of these, so best leave it for now for consistency, but may want to switch to single underscore in a follow-up if there's not some strong reason to dunder here